stdlib/public/core/UnsafeRawPointer.swift

```diff
@@ -1371,6 +1454,19 @@ public struct UnsafeMutableRawPointer: _Pointer {
     _debugPrecondition(
       byteCount >= 0, "UnsafeMutableRawPointer.copyMemory with negative count")
 
+    _uncheckedCopyMemory(from: source, byteCount: byteCount)
+  }
+
+  @_alwaysEmitIntoClient
+  internal func _uncheckedCopyMemory(
+    from source: UnsafeRawPointer,
+    byteCount: Int
+  ) {
+    _internalInvariant(
+      byteCount >= 0,
+      "UnsafeMutableRawPointer.copyMemory with negative count"
+    )
+
     _memmove(dest: self, src: source, size: UInt(byteCount))
```

Comment:
`assumeNonNegative` could come handy here, too. (Or it could induce dragons-from-nose problems.)